addons/web/__manifest__.py

```diff
@@ -23,6 +23,7 @@
     ],
     'qweb': [
         "static/src/xml/base.xml",
+        'static/src/xml/notification.xml',
```

Comment:
The list here is sorted alphabetically... keep it like that. Also, as all other files here use `"`, use it too instead of `'`